px4_ros2_cpp/test/unit/utils/geometry.cpp

```diff
@@ -8,6 +8,22 @@
 #include <gtest/gtest.h>
 #include <px4_ros2/utils/geometry.hpp>
 
+using namespace px4_ros2::literals;
```

Comment:
clang-tidy is a bit overly cautious here
```suggestion
using namespace px4_ros2::literals; // NOLINT
```